pkg/cri/server/podsandbox/controller.go

```diff
@@ -66,26 +64,26 @@ func init() {
 				return nil, fmt.Errorf("unable to init client for podsandbox: %w", err)
 			}
 
-			// Get base CRI dependencies.
-			criBasePlugin, err := ic.GetByID(plugins.InternalPlugin, "cri")
+			// Get runtime service.
+			criRuntimePlugin, err := ic.GetByID(plugins.CRIServicePlugin, "runtime")
 			if err != nil {
 				return nil, fmt.Errorf("unable to load CRI service base dependencies: %w", err)
```

Comment:
Can we change the log line from "CRI service base" to "CRI runtime service plugin"?